src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball extends Random {
```

Comment:
ball has no common behaviour and data with random
```suggestion
public class Ball {
```